lib/serverless/api-gateway.js

```diff
@@ -106,38 +106,45 @@ function isLambdaProxyEvent(event) {
   return isGatewayV1Event(event) || isGatewayV2Event(event)
 }
 
-function isGatewayV1Event(event) {
-  let result = false
-
-  if (event?.version === '1.0') {
-    result = true
-  } else if (
-    typeof event?.path === 'string' &&
-    (event.headers ?? event.multiValueHeaders) &&
-    typeof event?.httpMethod === 'string'
-    // eslint-disable-next-line sonarjs/no-duplicated-branches
-  ) {
-    result = true
-  }
+const v1Keys = [
```

Comment:
I can't find any information on what AWS does when some field from this list is not included in the payload. Does it send with `field: null`? Or does it completely omit the field. We are assuming that it sends `field: null` with this implementation, because the v1 example shows some such fields.